src/views/Lessons/Lessons.tsx

```diff
@@ -19,7 +26,48 @@ export const Lessons: FC = () => {
         heading="Simulator lessons"
         description="The simulator has been produced by a group of software engineers at Bloomberg in association with FIRST UK. It's simple solve every."
       />
-      <UnderConstruction title="Lessons page under construction" />
+      <Grid>
+        <LessonCard
+          mascotType="lesson-1"
+          subtitle="Lesson 1"
+          title="Motors"
+          description="Dolores et rerum hic quos sed at. Eos magnam laboriosam est repellat non. Dolores nihil suscipit quod voluptate."
+          call2actionText="View Challenges"
+          call2actionDest="../resources"
+        />
+        <LessonCard
+          mascotType="lesson-2"
+          subtitle="Lesson 2"
+          title="Distance sensors"
+          description="Dolores et rerum hic quos sed at. Eos magnam laboriosam est repellat non. Dolores nihil suscipit quod voluptate."
+          call2actionText="View Challenges"
+          call2actionDest="../resources"
+        />
+        <LessonCard
+          mascotType="lesson-3"
+          subtitle="Lesson 3"
+          title="Advanced driving"
+          description="Dolores et rerum hic quos sed at. Eos magnam laboriosam est repellat non. Dolores nihil suscipit quod voluptate."
+          call2actionText="View Challenges"
+          call2actionDest="../resources"
+        />
+        <LessonCard
+          mascotType="lesson-4"
+          subtitle="Lesson 4"
+          title="Colour sensor"
+          description="Dolores et rerum hic quos sed at. Eos magnam laboriosam est repellat non. Dolores nihil suscipit quod voluptate."
+          call2actionText="View Challenges"
+          call2actionDest="../resources"
+        />
+        <LessonCard
+          mascotType="lesson-5"
+          subtitle="Lesson 5"
+          title="Against the clock"
+          description="Dolores et rerum hic quos sed at. Eos magnam laboriosam est repellat non. Dolores nihil suscipit quod voluptate."
+          call2actionText="View Challenges"
+          call2actionDest="../resources"
+        />
```

Comment:
nit: would be good to have a json array object

```ts
const lessons = [
  {
    bgUrl,
    subtitle,
    title,
    description,
    call2actionText,
    call3actionDest,
  },
  ...
]

return <>
  ...
  {lessons.map((lesson, key) => <Lesson key={key} {...lesson} />)}
</>
```
so it can seamlessly migrate to a data persistent layer later.

However this is nit. feel free to address it in a future time.